functests/4_latency/latency.go

```diff
@@ -348,22 +353,34 @@ func extractLatencyValues(testName string, exp string, node *corev1.Node) string
 	return latencies
 }
 
-// checkMaximumEnvironLatencyValue should look for one of the following environment variables:
+// setMaximumLatencyValue should look for one of the following environment variables:
 // OSLAT_MAXIMUM_LATENCY: the expected maximum latency for all buckets in us
 // CYCLICTEST_MAXIMUM_LATENCY: the expected maximum latency for all buckets in us
 // HWLATDETECT_MAXIMUM_LATENCY: the expected maximum latency for all buckets in us
-func checkMaximumEnvironLatencyValue(testName string) error {
+// MAXIMUM_LATENCY: unified expected maximum latency for all tests
+func setMaximumLatencyValue(testName string) error {
 	if testName != strings.ToUpper(oslatTestName) &&
 		testName != strings.ToUpper(cyclictestTestName) &&
 		testName != strings.ToUpper(hwlatdetectTestName) {
 		return fmt.Errorf("testName variable has incorrect value %q", testName)
 	}
 
+	var err error
+	unifiedMaxLatencyEnv := os.Getenv("MAXIMUM_LATENCY")
+	if unifiedMaxLatencyEnv != "" {
+		if maximumLatency, err = strconv.Atoi(unifiedMaxLatencyEnv); err != nil {
```

Comment:
in general (let's not change this just yet) we should not set non-local variables by side effect. A better way:
```
maximumLatency, err = strconv.Atoi(unifiedMaxLatencyEnv)
if err != nil {
    ...
}
```